dt-core/global-functions.php

```diff
@@ -573,6 +573,73 @@ function dt_field_enabled_for_record_type( $field, $post ){
         return false;
     }
 
+    /**
+     * Render initial fields html to be displayed during bulk post type record creation.
+     *
+     * @param $dt_post_type
+     *
+     * @return false|string
+     */
+    function get_rendered_new_bulk_record_fields_html( $dt_post_type ) {
```

Comment:
@kodinkat this looks like a specific function for the bulk feature and not a global function